go/libkb/env.go

```diff
@@ -227,7 +227,14 @@ func (e *Env) GetMountDir() (string, error) {
 		func() string { return os.Getenv("KEYBASE_MOUNTDIR") },
 		func() string { return e.GetConfig().GetMountDir() },
 		func() string {
-			return filepath.Join(e.GetDataDir(), "fs")
+			switch runtime.GOOS {
```

Comment:
Ah good point, that's a bug here.  Will fix.

Reply:
Ah I'm realizing I was wrong saying "not from the changes in this PR" since this doesn't apply in the Linux case (or the macOS case before) since the data dir already has the `staging`/`devel` suffix when needed. Looks good now!